test/test_pow_df.cpp

```diff
@@ -0,0 +1,867 @@
+///////////////////////////////////////////////////////////////
+//  Copyright 2011 John Maddock.
```

Comment:
Sorry for not seeing it, but how does this test differ from test_pow.cpp?

Reply:
Hi John thanks for looking. The (new) test known as `test_pow_df.cpp` has parsed the original test and thrown away numbers that are so large that the arguments will definitely overflow `cpp_double_fp_backend<double>`.

So instead of having about 650 listed cases and controls, there remain about 650.